src/components/Icons/Twemoji/Mountain.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const Mountain = props => (
+  <svg viewBox="0 0 45 45" width={45} height={45} {...props}>
+    <g fill="none">
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)